INSTALL.md

```diff
@@ -309,8 +309,25 @@ Run [Yarn](https://www.yarnpkg.com/) to install all the required dependencies:
 yarn install
 ```
 
+## Manual setup for Arch linux
+
+The setup is the same as for Fedora, but the naming of packages might differ.
+Since Porta needs ImageMagick6, you need to install [libmagick6](https://aur.archlinux.org/packages/libmagick6/) from AUR.
+
+When running `bundle install`, you need to specify `PKG_CONFIG_PATH`.
+```shell
+PKG_CONFIG_PATH=/usr/lib/imagemagick6/pkgconfig bundle install
```

Comment:
Can use linuxbrew (homebrew) and be done with different distribution of Linux?
I am worried of the number of platform we are going to officially support for development

This could be put in Troubleshooting in wiki, and if it is up to date then good if not then "best customer support"

Reply:
I am not sure if it can be done with different distributions of Linux. But this is a solution when you need to have more versions of ImageMagick on your system.

I agree that this might be more suitable for Troubleshooting in the wiki. Do we have something like that somewhere?

Reply:
Wiki was opened few months ago, go ahead if you want to create a page